python/nano/test/openvino/pytorch/test_openvino.py

```diff
@@ -81,3 +82,27 @@ def test_openvino_inputsample_from_trainloader(self):
         trainer.fit(pl_model, dataloader)
         assert pl_model.forward == pl_model._torch_forward 
         assert  pl_model.ov_infer_engine is None
+
+    def test_trainer_trace_openvino(self):
+        trainer = Trainer(max_epochs=1)
+        model = mobilenet_v3_small(num_classes=10)
+        
+        x = torch.rand((10, 3, 256, 256))
+        y = torch.ones((10, ), dtype=torch.long)
+
+        # trace a torch model 
+        openvino_model = trainer.trace(model, x, 'openvino')
```

Comment:
`trace` without `compile`?

Reply:
Yes, `compile` should be a function only for training after we refactored all other modules.   
`trainer.quantize` `trainer.trace` should work on any instance of nn.Module including LightningModule.

Reply:
As discussed, `trainer.compile(model, loss, ..., openvino=True, onnx=False, quanti=False)` will finally become `trainer.compile(model, loss, ...)`. Seems that `compile` is then only for converting a pytorch model into PL for training then.